system-contracts/contracts/interfaces/IBaseToken.sol

```diff
@@ -17,15 +17,15 @@ interface IBaseToken {
 
     function mint(address _account, uint256 _amount) external;
 
-    function withdraw(address _l1Receiver) external payable;
+    function withdraw(bytes calldata _l1Receiver) external payable;
```

Comment:
P2PKH => 20-byte
Bech32 => 32-byte 
Bech32m => 32-byte 

based on this information, maybe we could use `byte32` instead of `bytes`

`bytes`  allowing user to give their arbitrary length information as input 

cc: @0xatomFusion

Reply:
This is a valid point!
I'll introduce the modification in the next PR